src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -62,6 +62,45 @@
 short-summary: Manage Azure Management Groups.
 """
 
+helps['account management-group tenant-backfill'] = """
+type: group
+short-summary: Backfill Tenant Subscription Operations for Management Groups
+"""
+
+helps['account management-group tenant-backfill get'] = """
+type: command
+short-summary: Get the backfill status for a tenant.
+long-summary: Get the backfill status for a tenant.
```

Comment:
`long-summary` should provide additional information, instead of being a duplicate of `short-summary`. See https://github.com/Azure/azure-cli/pull/6426#discussion_r1391924773.